src/platform/forms-system/src/js/fields/ObjectField.jsx

```diff
@@ -180,7 +180,10 @@ class ObjectField extends React.Component {
 
     // description and title setup
     const showFieldLabel = uiOptions.showFieldLabel;
-    const fieldsetClassNames = uiOptions.classNames;
+    const fieldsetClassNames = classNames(uiOptions.classNames, {
+      'vads-u-margin-bottom--2': !environment.isProduction(),
+    });
```

Comment:
From [the usage docs](https://www.npmjs.com/package/classnames) on `classNames`:

```js
// other falsy values are just ignored
classNames(null, false, 'bar', undefined, 0, 1, { baz: null }, ''); // => 'bar 1'
```

So it should be ok if `uiOptions.classNames` is undefined.

Reply:
Nice. Thank you!